### DISCUSSION TITLE
There's no way of seeing how many ranked finished plays there are on the map in lazer and osu!web (same thing with score differential between leaderboard entries)

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#29416] [Ideas] [answered]
On stable you can see how many people finished the map if you finished it, as well as you can see how much more score #1 has over #2, etc.
![image](https://github.com/user-attachments/assets/a1e0366d-2421-4c26-b45b-41c3ee6982ec)

You can't see it anywhere on Lazer, nor on osu!web
![osu_2024-08-14_13-18-58](https://github.com/user-attachments/assets/74851ab5-547b-4f1f-b5e0-9710ad559836)
![image](https://github.com/user-attachments/assets/7115b0dc-acdf-4c24-83a0-670cdf09a361)
![image](https://github.com/user-attachments/assets/b183d98f-34d3-43fd-ba77-f208a9501c13)

I think it's a good metric to keep, because you can tell with that how many users can finish the map, and where in this pile of users you are placed.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/discussions/28926
